src/databricks/labs/ucx/source_code/known.json

```diff
@@ -26274,6 +26274,25 @@
   "tomlkit": {
     "tomlkit": []
   },
+  "toolz": {
+    "tlz": [],
```

Comment:
Same question about the `tlz` not matching `toolz` as in other Pr

Reply:
We don't control this. The pypi package is `toolz`, it contains a `tlz` module, see https://github.com/pytoolz/toolz/

Reply:
yes, I understand that, but want to confirm that our logic will still work when this is the case

Reply:
this was tested by running `make solacc` so I confirm